src/Bswup/README.md

```
@@ -133,37 +204,204 @@ The most important line here is the last line which is the only mandatory config
 - `ignoreDefaultExclude`: Ignores the default asset **excludes** array which is provided by the Bswup itself which is like this: 
     ```js
-    [/^_content\/Bit\.Bswup\/bit-bswup\.sw\.js$/, /^service-worker\.js$/]
+    [
+        /^_content\/Bit\.Bswup\/bit-bswup\.sw\.js$/,
+        /^_content\/Bit\.Bswup\/bit-bswup\.sw\.min\.js$/,
+        /^_content\/Bit\.Bswup\/bit-bswup\.sw-cleanup\.js$/,
+        /^_content\/Bit\.Bswup\/bit-bswup\.sw-cleanup\.min\.js$/,
+        /^service-worker\.js$/,
+    ]
     ```
     #### Keep in mind that caching service-worker related files will corrupt the update cycle of the service-worker. Only the browser should handle these files. 
```

markdownlint-cli2 (0.23.0)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)